libvast/include/vast/concept/parseable/vast/option_set.hpp

```diff
@@ -43,51 +46,57 @@ struct option_set_parser : parser_base<option_set_parser> {
   }
   template <class Attribute>
   bool parse_short_form(auto& f, const auto& l, Attribute& x) const {
-    static constexpr auto short_form_parser
-      = '-' >> (parsers::alpha) >> ignore(+whitespace) >> parsers::data;
-    auto short_form_opt = std::tuple<char, vast::data>{};
+    static constexpr auto short_form_key_parser = '-' >> (parsers::alpha);
+    char short_form_opt;
```

Comment:
I think this should be initialized. This is a nice advantage of auto xx = T {}. With the usage of auto at the begining there is no way to forget about it :P

Reply:
Initialized in https://github.com/tenzir/vast/pull/2877/commits/ab0b8f1dca9c770e41d2f57f47cb087697b29d56